app/models/ingredient_mention.rb

```diff
@@ -0,0 +1,3 @@
+class IngredientMention < ApplicationRecord
+	validates :name, uniqueness: true
+end
```

Comment:
Shouldn't this belongs_to movie and ingredient? As well, shouldn't ingredient and movie has_many ingredient_mentions?